lib/has_tag_string/has_tag_string.rb

```diff
@@ -135,7 +135,8 @@ def get_tag_values(tag_as_string)
 
     # Adds a new tag to the model, if it isn't already there
     def add_tag_if_not_already_present(tag_as_string)
-      self.tag_string = self.tag_string + " " + tag_as_string
+      return tag_string if tag_string.include?(tag_as_string)
```

Comment:
This would prevent `foo` being added if `foobar` is already present. Real world example would be adding `NHS` if `NHSChoicesID` was present

Reply:
Ah good catch! Fixed in 5c265d33d.